layouts/partials/footer.html

```diff
@@ -1,14 +1,14 @@
 {{- $s := .Site.Params }}
-{{ $t := .Site.Title }}
+{{- $c := or .Site.Copyright .Site.Title }}
```

Comment:
@iain-henderson,

The `or` function doesn't return a boolean? I think a better approach here would be

```
{{- $c := default .Site.Title .Site.Copyright  }}
```

Reply:
That makes sense, I've updated it.